ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__automated-release-stable-4.16-upgrade-from-stable-4.16.yaml

```diff
@@ -57,6 +57,16 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: aws-ipi-f999
```

Comment:
@rioliu-rh should we add these jobs or update the existing ones?

Reply:
I suggest to do clean up for old jobs, and add new jobs with typical profiles